apps/discord-bot/src/hook/ready.ts

```diff
@@ -0,0 +1,10 @@
+import { Events } from 'discord.js'
+
+import { CUGetReg } from '../core/CUGetReg'
+import { HookFunction } from './hook'
+
+export const ReadyHook: HookFunction = (client: CUGetReg): void => {
+  client.once(Events.ClientReady, async () => {
+    console.log(`${client.user.username} Online!`)
```

Comment:
this logs when this bot is online right?

Reply:
Right